### PR TITLE
[Core] [Docs] Remove travis-ci references

### PR DESCRIPTION
**Pull request by Spaceman**
## Description
"The migration already happened, and it broke CI for us. You might have noticed there aren't any builds on PRs anymore. All the Travis stuff should actually be removed altogether." - @fauxpark 

## Types of Changes

- [X] Core
- [X] Documentation

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: docs/fr-fr/contributing.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Les PR de nouvelles fonctionnalités de de correction de bug affectent tous les
 Voici quelques choses à garder en tête lorsque vous travaillez sur une fonctionnalité ou un bug fix.
 
 * **Désactivé par défaut** - la mémoire est plutôt limitée sur la plupart des puces que QMK supporte, et il est important que les keymaps courantes ne soient pas cassées. S'il vous plaît faites que vos features doivent être **activées** plutôt que désactivées. Si vous pensez qu'elle devrait être activée par défaut, ou que cela réduit la taille du code, parlez-nous-en.
-* **Compilez localement avant de soumettre** - Cela devrait aller sans dire, mais votre code doit compiler! Notre système Travis devrait relever les problèmes, mais il est généralement plus rapide de compiler quelques claviers en local plutôt que d'attendre le retour des résultats
+* **Compilez localement avant de soumettre** - Cela devrait aller sans dire, mais votre code doit compiler! <!--FIXME: We should have more info here but I don't know french -->
```
Comment (Spaceman) [inline docs/fr-fr/contributing.md:137]:
```suggestion
* **Compilez localement avant de soumettre** - Cela devrait aller sans dire, mais votre code doit compiler! Vous devriez toujours faire gaffe à ce que vos changements compilent avant d'ouvrir une pull request.
```

Comment (Spaceman) [reply docs/fr-fr/contributing.md:137]: Translation from native speaker

File: docs/ja/contributing.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ enum my_keycodes {
 機能やバグの修正に取り組む時に留意すべき幾つかの事があります。
 
 * **デフォルトで無効** - QMK がサポートするほとんどのチップでメモリがかなり制限されており、現在のキーマップが壊れていないことが重要です。ですので、あなたの機能をオフにするのではなく**オン**にするようにしてください。デフォルトでオンにすべき場合、あるいはコードのサイズを小さくする必要がある場合は、相談してください。
-* **サブミットする前にローカルでコンパイル** - これが明白であることを願っていますが、コンパイルする必要があります。私たちの Travis システムは全ての問題をキャッチしますが、結果が返ってくるのを待つ代わりに幾つかのキーボードをローカルでコンパイルする方が一般的に速いです。
+* **サブミットする前にローカルでコンパイル** - これが明白であることを願っていますが、コンパイルする必要があります。<!--FIXME: We should have more info here but I don't know japanese -->
```
Comment (Spaceman) [inline docs/ja/contributing.md:156]:
```suggestion
* **サブミットする前にローカルでコンパイル** - これが明白であることを願っていますが、コンパイルする必要があります。プルリクエストを作成する前に、変更した内容がコンパイルできるかどうかを常に確認する必要があります。
```
According to DeepL this is a good translation, but I'd like to get a native speaker to confirm.

File: docs/fr-fr/contributing.md (hunk 1 above)
Comment (ASH-ART) [reply docs/fr-fr/contributing.md:137]: I confirm that the translation is good.

File: docs /zh-cn /README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
 # QMK机械键盘固件
```
Comment (zvecr) [inline docs /zh-cn /README.md:1]: This file should be renamed back, removing the added space.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: docs /zh-cn /README.md (hunk 3 above)
Comment (Spaceman) [reply docs /zh-cn /README.md:1]: Not sure how that happened ill try to fix it

**Review by tzarc — APPROVED** [review]

File: docs/ja/contributing.md (hunk 2 above)
Comment (fauxpark) [reply docs/ja/contributing.md:156]: @mtei ?

Comment (mtei) [reply docs/ja/contributing.md:156]: Looks good to me !

@umi-umi @shelaf @s-show よかったらレビューお願いします。

**Review by zvecr — APPROVED** [review]

**Pull request merged**